lab1/src/task2/Internship.java

```diff
@@ -0,0 +1,28 @@
+package task2;
+
+import java.util.List;
+
+public class Internship {
+    public String name;
+    public String minGrade;
+    public List<Student> students;
+
+    public Internship(String _name, String _minGrade, List<Student> _students) {
```

Comment:
Don't use identifiers starting with _ (it's considered bad coding style).  If you want to find out more, see https://www.ibm.com/docs/en/zos/2.3.0?topic=tokens-identifiers